foundation/templates/includes/membership_list.html

```diff
@@ -0,0 +1,34 @@
+<div class="web-list-item membership-list-item">
```

Comment:
this can now go in `foundation/doctype/membership/templates/membership_row.html`